main/src/com/google/refine/expr/functions/ToDate.java

```diff
@@ -76,6 +78,8 @@ public Object call(Properties bindings, Object[] args) {
                 o1 =  ((Long) arg0).toString(); // treat integers as years
```

Comment:
Probably should update or remove these comments

Reply:
Don't remove!  Update!  We actually need more comments in our code base to help explain more for new contributors coming in and make it more attractive to join in helping.